### PR TITLE
Allow picking fields to exclude for executions.get

### PR DESCRIPTION
**Pull request by enykeev**

### CODE REVIEW

File: packs/st2/actions/lib/action.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def _get_auth_token(self):
 
         return token
 
-    def _run_client_method(self, method, method_kwargs, format_func):
+    def _run_client_method(self, method, method_kwargs, format_func, format_kwargs={}):
```
Comment (Kami) [inline packs/st2/actions/lib/action.py]: Should default to a non mutable type (e.g. None and do format_kwargs = format_kwargs or {} in the body)/

**Comment by Kami** [comment]
LGTM, +1

File: packs/st2/actions/lib/action.py (hunk 1 above)
Comment (manasdk) [inline packs/st2/actions/lib/action.py]: Like Kami said - 
https://pythonconquerstheuniverse.wordpress.com/2012/02/15/mutable-default-arguments/

Comment (enykeev) [inline packs/st2/actions/lib/action.py]: yeah, forgot about that. Thanks!

**Pull request merged**